src/main/java/seedu/address/model/person/TelegramHandle.java

```diff
@@ -11,13 +11,14 @@ public class TelegramHandle {
 
     public static final TelegramHandle NULL_TELEGRAM_HANDLE = null;
     public static final String MESSAGE_CONSTRAINTS = "Telegram Handle must "
-            + "use a-z, 0-9 and underscores. Minimum length is 5 characters";
+            + "use a-z, 0-9 and underscores. Minimum length is 5 characters. It must start with a letter and cannot "
+            + "end with an underscore";
 
     /*
      * The telegram handle uses a-z, 0-9 and underscores.
      * Minimum length is 5 characters.
      */
-    public static final String VALIDATION_REGEX = "^[a-zA-Z0-9_]{5,32}$";
+    public static final String VALIDATION_REGEX = "^[a-zA-Z][a-zA-Z0-9_]{3,28}[a-zA-Z0-9]$";
```

Comment:
Is there an explanation for the upper bound on 28? Is the maximum handle length 30? Explain in the comment above for the regex.

Reply:
Updated thanks

Reply:
max is 32 characters, my bad for not specifying